src/analytics/private/sqls/redshift/sp-scan-metadata.sql

```diff
@@ -0,0 +1,391 @@
+	FROM (
+		SELECT 
+			event_name, 
+			project_id, 
+			app_info_app_id, 
+			property_category, 
+			property_name, 
+			value_type, 
+			platform,
+			'[' || LISTAGG(property_value, ', ') WITHIN GROUP (ORDER BY property_value) || ']' AS property_values
+		FROM (
+      SELECT 
+        event_name, 
+        project_id, 
+        app_info_app_id, 
+        property_category, 
+        property_name, 
+        value_type,
+        property_value,
+          '[' || LISTAGG(platform, ', ') WITHIN GROUP (ORDER BY platform) || ']' AS platform
+      FROM (
+        SELECT 
+          event_name, 
+          project_id, 
+          app_info_app_id, 
+          property_category, 
+          property_name, 
+          property_value, 
+          value_type, 
+          platform, 
+          parameter_count
+        FROM (
+          SELECT 
+            event_name, 
+            project_id, 
+            app_info_app_id, 
+            property_category, 
+            property_name, 
+            property_value, 
+            value_type, 
+            platform, 
+            parameter_count,
+              ROW_NUMBER() OVER (PARTITION BY event_name, project_id, app_info_app_id, property_category, property_name ORDER BY parameter_count DESC) AS row_num
+          FROM (
+            SELECT 
+              event_name, 
+              project_id, 
+              app_info_app_id, 
+              property_category, 
+              property_name, 
+              property_value, 
+              platform, 
+              value_type, 
+              count(*) AS parameter_count 
+            FROM properties_temp_table
+            GROUP BY event_name, project_id, app_info_app_id, property_category, property_name, property_value, value_type, platform
+          )
+        )
+        WHERE 
+          row_num <= top_frequent_properties_limit OR
+          (event_name = '_page_view' AND property_name IN ('_page_title', '_page_url')) OR
+          (event_name = '_screen_view' and property_name IN ('_screen_name', '_screen_id'))
+        ORDER BY event_name, project_id, app_info_app_id      
+			)
+			GROUP By event_name, project_id, app_info_app_id, property_category, property_name, value_type, property_value
+		)
+		GROUP By event_name, project_id, app_info_app_id, property_category, property_name, value_type, platform
+	);
+
+	CALL {{schema}}.{{sp_clickstream_log}}(log_name, 'info', 'Insert all parameters data into event_properties_metadata table successfully.');	
+
+  -- user attribute
+	INSERT INTO {{schema}}.event_properties_metadata (id, type, prefix, event_name, project_id, app_id, category, metadata_source, property_type, property_name, property_id, value_type, value_enum, platform) 
+	SELECT
+		project_id || '#' || app_info_app_id || '#' || event_name || '#' || property_name AS id,
+		'USER_ATTRIBUTE#' || project_id || '#' || app_info_app_id || '#' || property_name AS type,
+		'USER_PARAMETER#' || project_id || '#' || app_info_app_id AS prefix,
```

Comment:
`USER_PARAMETER` should be `USER_ATTRIBUTE`

Reply:
Fixed